### PR TITLE
fix summary in mathics.builtin.specialfns

### PR DESCRIPTION
**Pull request by mmatera**
This PR contains another round of polishing Builtin's summary_text 

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/240"></a>

### CODE REVIEW

File: mathics/builtin/specialfns/othogonal.py [hunk 1]
```diff
@@ -1,294 +0,0 @@
-"""
```
Comment (rocky) [inline mathics/builtin/specialfns/othogonal.py:1]: Why is this removed? It is listed in https://reference.wolfram.com/language/guide/SpecialFunctions.html

**Comment by rocky** [comment]
Looks fine other than the fact that Orthogonal Polynomials have been removed.

File: mathics/builtin/specialfns/othogonal.py (hunk 1 above)
Comment (mmatera) [reply mathics/builtin/specialfns/othogonal.py:1]: Actually, was renamed to *orthogonal*

File: mathics/builtin/specialfns/bessel.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ class AiryAiPrime(_MPMathFunction):
         "Derivative[1][AiryAiPrime]": "(#1 AiryAi[#1])&",
     }
 
-    summary_text = "Derivative of the Airy function Ai"
+    summary_text = "Derivative of the Airy's function Ai"
```
Comment (TiagoCavalcante) [inline mathics/builtin/specialfns/bessel.py:82]: Capital letter here and in other places.

Comment (mmatera) [reply mathics/builtin/specialfns/bessel.py]: fixed

File: mathics/builtin/specialfns/expintegral.py [hunk 3]
```diff
@@ -61,6 +63,7 @@ class ProductLog(_MPMathFunction):
      = -Graphics-
     """
 
+    summary_text = "Lambter's W function"
```
Comment (TiagoCavalcante) [inline mathics/builtin/specialfns/expintegral.py:66]: Lambter -> Lambert

**Comment by rocky** [comment]
LGTM
